cmd/earthly/satellite_cmds.go

```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 	"time"
 
+	"github.com/jedib0t/go-pretty/table"
```

Comment:
is there a reason we can't use `"text/tabwriter"` instead?

we already use it in many of our commands, see  https://github.com/earthly/earthly/blob/main/cmd/earthly/account_cmds.go#L442-L443 for an example.

Reply:
It takes a lot more code, but I can rewrite it if you want

Reply:
Rewritten

Reply:
thanks! I think it's useful to stay consistent with how we write out these tables.